namelists_bios/blaze.nml

```diff
@@ -0,0 +1,11 @@
+&blazenml
```

Comment:
okay - this has been updated by Lars/Yohanna I think after I have branched off but before you branched off